lib/teamlab/modules/people/people_remove.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Teamlab
+  # Module for people removing process
+  module PeopleRemove
+    # Start a remove process
+    # @param userId [String] guid of user to remove
+    # @param [Hash] remove progress data
+    def start_remove(user_id)
+      @request.post(%w[remove start], userId: user_id)
+    end
+
+    # Returns the progress of the started remove process
+    # @param userId [String] guid of user to remove
+    # @param from_user_id [String] remove progress data
+    def get_remove_progress(user_id)
```

Comment:
In general - better not to name methods starting with `get`

Maybe `def remove_progress(user_id)`

Reply:
<img width="225" alt="Screen Shot 2021-08-16 at 17 21 30" src="https://user-images.githubusercontent.com/40513035/129578820-82b117f4-d9d3-4ad0-bd40-fe52f1578907.png">
Changed it but that's how this method is named in the documentation

Reply:
I think we do not need to duplicate documentation name by 100%

Ading `get` is unusual for `ruby` language, not for any language
But we're writing ruby library so I think we should first of all try to be in order with ruby language guidelines, not with strange names in our documentation, like `photoes`))